src/sidebar/components/Annotation/AnnotationEditor.js

```diff
@@ -43,63 +47,86 @@ function AnnotationEditor({
   );
 
   const store = useStoreProxy();
-  const draft = store.getDraft(annotation);
   const group = store.getGroup(annotation.group);
 
-  if (!draft) {
-    // If there's no draft, we can't be in editing mode
-    return null;
-  }
-
   const shouldShowLicense =
     !draft.isPrivate && group && group.type !== 'private';
 
   const tags = draft.tags;
   const text = draft.text;
   const isEmpty = !text && !tags.length;
 
-  const onEditTags = ({ tags }) => {
-    store.createDraft(draft.annotation, { ...draft, tags });
-  };
+  const onEditTags = useCallback(
```

Comment:
The fact that `onEditTags` takes an object with a `tags` property rather than just the `tags` value as an argument is, I believe, a vestige from the AngularJS => Preact migration. We might as well fix that. Same with `onEditText`.